compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1230,6 +1231,8 @@ unsafe extern "C" {
     pub(crate) fn LLVMIsAInstruction(Val: &Value) -> Option<&Value>;
     pub(crate) fn LLVMGetFirstBasicBlock(Fn: &Value) -> &BasicBlock;
     pub(crate) fn LLVMGetOperand(Val: &Value, Index: c_uint) -> Option<&Value>;
+    pub(crate) fn LLVMGetNextInstruction(Val: &Value) -> Option<&Value>;
+    pub(crate) fn LLVMInstructionEraseFromParent(Val: &Value);
```

Comment:
any of these new functions safe to call? So that we could mark them as safe?

Reply:
GetNextInstructions seems risky (vaguely wave at iterator invaldiation).

@kevinsala Can you think of ways to cause UB when using any of these three functions in an LLVM build without assertions? I _think_ Erase/Delete should be save:
If you don't pass the right type (instruction or function) that should be noticed during the unwrapping, or? If we try to delete something that still has users, a check will catch it. Can you think of other ways?